src/main/java/org/folio/circulation/CirculationVerticle.java

```diff
@@ -73,6 +74,8 @@ public void start(Future<Void> startFuture) {
       "/circulation/rules/notice-policy-all",
         client)
         .register(router);
+    new ScheduledNoticeProcessingResource(client).register(router);
```

Comment:
Please include a blank line above this to separate it from the previous definition

Reply:
Done